source/01-LE/02.ptx

```diff
@@ -73,35 +73,179 @@
 </definition>
 
 <activity estimated-time='10'>
-    <!-- TODO split into activities per operation, have students perform op on example -->
-    <statement>
+    <introduction>
         <p>
-  Following are seven procedures used to manipulate an augmented matrix.
-  Label the procedures that would result in an equivalent augmented
-  matrix as <em>valid</em>, and label the procedures that might
-  change the solution set of the corresponding linear system as
-  <em>invalid</em>.
+Consider whether these matrix manipulations (A) <em>must keep</em> or
+(B) <em>could change</em>
+the solution set for the corresponding linear system.
         </p>
-    <ol marker="A.">
-<li> Swap two rows.
-      </li>
-<li> Swap two columns.
-      </li>
-<li> Add a constant to every term in a row.
-      </li>
-<li> Multiply a row by a nonzero constant.
-      </li>
-<li> Add a constant multiple of one row to another row.
-      </li>
-<li> Replace a column with zeros.
-      </li>
-<li> Replace a row with zeros.
-      </li>
-    </ol>
-  <!--
-    This activity could be ran as a card sort.  Allow 5 additional minutes for intra team discussion.
-  -->
-    </statement>
+    </introduction>
+    <task>
+        <statement>
+            <p>
+Swapping two rows, for example:
+            </p>
+<sidebyside>
+    <p>
+        <me>
+            \left[\begin{array}{cc|c} 1 &amp; 2 &amp; 3 \\ 4 &amp; 5 &amp; 6 \end{array}\right]
+        \sim
+            \left[\begin{array}{cc|c} 4 &amp; 5 &amp; 6 \\ 1 &amp; 2 &amp; 3 \end{array}\right]
+        </me>
+    </p>
+    <p>
+        <md>
+            <mrow>x  +2y \amp= 3 \amp 4x + 5y \amp= 6</mrow>
+            <mrow>4x + 5y \amp= 6 \amp x  +2y \amp= 3 </mrow>
+        </md>
+    </p>
+</sidebyside>
+        </statement>
+    </task>
+    <task>
+        <statement>
+            <p>
+Swapping two columns, for example:
+            </p>
+<sidebyside>
+    <p>
+        <me>
+            \left[\begin{array}{cc|c}  1 &amp; 2 &amp; 3 
+                                    \\ 4 &amp; 5 &amp; 6 \end{array}\right]
+        \sim
+            \left[\begin{array}{cc|c} 2 &amp; 1 &amp; 3 
+                                    \\ 5 &amp; 4 &amp; 6 \end{array}\right]
+        </me>
+    </p>
+    <p>
+        <md>
```

Comment:
Should we ask the students to write the systems of equations?

Reply:
Not sure if generated a notification, but that was specifically a question for @StevenClontz .

Reply:
My inclination is "no" - that's the LE1 outcome, and I like having this activity be about discussion more than writing/manipulation, so providing the matrix + corresonding system together gets that going.

Reply:
If you feel strongly otherwise, I'd consider providing the first 3-4 systems, then having students write out the last 4-3.

Reply:
Sounds reasonable to me!

Reply:
Sorry, to be clear, providing it for the first n, and having them do it for the last few sounds reasonable to me.

Reply:
I like the opportunity to interleave, provided that (to your point) it doesn't increase the cognitive load to where they can't engage with the new activity.  But LE1 is pretty straightforward I think.